templates/firefox-ios.xliff

```diff
@@ -2006,6 +2050,17 @@
         <source>Troubleshoot</source>
         <note>Title of link to help page to find out how to solve Sync issues</note>
       </trans-unit>
+      <trans-unit id="Settings.WebsiteData.Button">
+        <source>Show More</source>
+        <note>Button shows all websites on website data tableview
```

Comment:
I commented in the previous PR about these comments.

Besides the request of only having one (if possible), how can they be so different? The first one (show all websites) is plausible, the other 3 not really for a "Show More" string. Are we using the wrong string ID in some places?

Reply:
Sorry didnt understand this earlier. I see the problem now